### ISSUE TITLE
BMW Stiftung dead link

### ISSUE THREAD

**Issue by matkoniecz** [renewablefreedom/funding-sources#9]
> BMW Stiftung,"""The BMW Foundation brings together leaders from across communities, cultures and countries to drive Social Innovation, promote Global Dialogue, and encourage Responsible Leadership. We are convinced that our approach breaks down barriers between politics, business and civil society and allows society to benefit from the creativity and diversity that result from cross-sector collaboration.“ See esp. Their „Social Entrepreneurship“-Program.","<a href=""http://www.bmw-stiftung.de/en/what-we-do/social-entrepreneurship/"">http://www.bmw-stiftung.de/en/what-we-do/social-entrepreneurship/</a>"

https://bmw-foundation.org/de/en/what-we-do/social-entrepreneurship died - I have not tried to recover it, so maybe it is just moved

**Comment by moba** [comment]
Yeah I think we can simply link to their main homepage. If people are interested, they will find the Social Entrepreneurship program from that pointer.
